touchforms/backend/localsettings.py.example

```diff
@@ -6,3 +6,14 @@ PERSISTENCE_DIRECTORY = "/path/to/www/data/touchforms"
 
 # from jython_ssl import SSLContext, TRUST_ALL_CONTEXT
 # SSLContext.setDefault(TRUST_ALL_CONTEXT)
+
+USES_POSTGRES = True
+POSTGRES_TABLE = "formplayer_session"
+POSTGRES_JDBC_JAR = "{COMMCARE_HOME}/submodules/touchforms-src/touchforms/backend/jrlib/postgresql-9.0-801.jdbc3.jar"
+
+DATABASE = {
+        'serverName': 'localhost:5432',
+        'databaseName': 'commcarehq',
+        'user': 'admin',
+        'password': '123',
```

Comment:
@czue the Java method we're calling requires the camel case and naming here

 https://github.com/dimagi/touchforms/pull/82/files#diff-b5812de1fa561253ef3c9042d4991a85R128